packages/server/src/integrations/rest.js

```diff
@@ -0,0 +1,116 @@
+const fetch = require("node-fetch")
+const { FIELD_TYPES, QUERY_TYPES } = require("./Integration")
+
+const SCHEMA = {
+  docs: "https://github.com/node-fetch/node-fetch",
+  datasource: {
+    url: {
+      type: FIELD_TYPES.STRING,
+      default: "localhost",
+      required: true,
+    },
+  },
+  query: {
+    create: {
+      type: QUERY_TYPES.FIELDS,
+      fields: {
+        path: {
+          type: FIELD_TYPES.STRING,
+        },
+        headers: {
+          type: FIELD_TYPES.OBJECT,
+        },
+        requestBody: {
+          type: FIELD_TYPES.JSON,
+        },
+      },
+    },
+    read: {
+      type: QUERY_TYPES.FIELDS,
+      fields: {
+        path: {
+          type: FIELD_TYPES.STRING,
+        },
+        headers: {
+          type: FIELD_TYPES.OBJECT,
+        },
+      },
+    },
+    update: {
+      type: QUERY_TYPES.FIELDS,
+      fields: {
+        path: {
+          type: FIELD_TYPES.STRING,
+        },
+        headers: {
+          type: FIELD_TYPES.OBJECT,
+        },
+        requestBody: {
+          type: FIELD_TYPES.JSON,
+        },
+      },
+    },
+    delete: {
+      type: QUERY_TYPES.FIELDS,
+      fields: {
+        path: {
+          type: FIELD_TYPES.STRING,
+        },
+        headers: {
+          type: FIELD_TYPES.OBJECT,
+        },
+        requestBody: {
+          type: FIELD_TYPES.JSON,
+        },
+      },
+    },
+  },
+}
+
+class RestIntegration {
+  constructor(config) {
+    this.config = config
+  }
+
+  async create({ path, headers = {}, json }) {
+    const response = await fetch(this.config.url + path, {
+      method: "POST",
+      headers,
+      body: JSON.stringify(json),
```

Comment:
Currently this only allows use of JSON based API, but in the videos you showed adding JSON headers like `Content-Type` - either might be good to hard-code JSON headers and make them non-editable so its obvious that it currently only supports JSON, or use the `Accept` header to work out whether to parse as JSON, XML, plaintext etc

Reply:
Good point - I considered adding an option to choose the "type" of the body/response, but at the moment it doesn't make a whole lot of sense to return anything other than JSON based on the fact that the schema gets built from an array of objects. 

I'll add some parsing based on the accept header so there's a bit more flexibility - that makes sense 👍